grafana/resource_playlist.go

```diff
@@ -74,27 +72,28 @@ func ResourcePlaylist() *schema.Resource {
 }
 
 func CreatePlaylist(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
-	client := meta.(*client).gapi
+	/*client := meta.(*client).gapi
```

Comment:
Playlist changes are temporary due to an unrelated breaking change in the golang client.